F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "FastJet_Julia_Wrapper"
+version = v"0.5.0"
+
+# Collection of sources required to build FastJet_Julia_Wrapper
+sources = [
+   GitSource("https://github.com/jstrube/FastJet_Julia_Wrapper.git", "1ed483a0449a5a7330e0d899492b5c5d2247d47a"; unpack_target="FastJet_Julia_Wrapper"),
+   FileSource("https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-x86_64-w64-mingw32.tar.gz", "9c58bc0873e52cf6c41108a7a2b100f68419478f10c6fe635197b1bf47eec64d"; unpack_target="x86_64-w64-mingw32"),
+   FileSource("https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-x86_64-linux-gnu.tar.gz", "34b6e59acf8970a3327cf1603a8f90fa4da8e5ebf09e6624509ac39684a1835d"; unpack_target="x86_64-linux-gnu"), 
+   FileSource("https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-x86_64-linux-gnu.tar.gz", "34b6e59acf8970a3327cf1603a8f90fa4da8e5ebf09e6624509ac39684a1835d"; unpack_target="x86_64-linux-musl"),
+   FileSource("https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-x86_64-apple-darwin14.tar.gz", "a9537f53306f9cf4f0f376f737c745c16b78e9cf635a0b22fbf0562713454b10"; unpack_target="x86_64-apple-darwin14")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+ln -s ${WORKSPACE}/srcdir/${target}/include/ /opt/${target}/${target}/sys-root/usr/local
```

Comment:
I think this should rather be something like
```sh
export CXXFLAGS="-I${WORKSPACE}/srcdir/${target}/include"
```

Reply:
If that works, great! I just tried random things based on the error message. Let me try...

Reply:
Ah, right. That was needed because the dependency needs that path. The error message is: 
```
[00:25:51] CMake Error in CMakeLists.txt:
[00:25:51]   Imported target "JlCxx::cxxwrap_julia" includes non-existent path
[00:25:51] 
[00:25:51]     "/opt/x86_64-linux-gnu/x86_64-linux-gnu/sys-root/usr/local/include"
[00:25:51] 
[00:25:51]   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
[00:25:51] 
[00:25:51]   * The path was deleted, renamed, or moved to another location.
[00:25:51] 
[00:25:51]   * An install or uninstall procedure did not complete successfully.
[00:25:51] 
[00:25:51]   * The installation package was faulty and references files it does not
[00:25:51]   provide.
```